sgv2-service-common/src/main/java/io/stargate/sgv2/common/cql/builder/QueryBuilderImpl.java

```diff
@@ -749,6 +750,18 @@ public Query build() {
     return query.build();
   }
 
+  @DSLAction
+  public BatchQuery buildForBatch() {
+    if (parameters != null) {
+      throw new IllegalStateException("Parameters aren't supported for a batched query");
+    }
+    BatchQuery.Builder query = BatchQuery.newBuilder().setCql(buildCql());
+    if (!generatedBoundValues.isEmpty()) {
+      query.setValues(Values.newBuilder().addAllValues(generatedBoundValues).build());
+    }
+    return query.build();
+  }
+
```

Comment:
Should we extend the `QueryBuilderTest`?

Reply:
Yes I can add a couple of tests.